src/lib/y2partitioner/widgets/blk_devices_table.rb

```diff
@@ -27,6 +29,16 @@ def handle
         @pager.handle("ID" => id)
       end
 
+      # Device object selected in the table
+      #
+      # @return [Y2Storage::BlkDevice, nil] nil if anything is selected
+      def selected_device
+        return nil if items.empty? || !value
+
+        device_name = value[/table:(.*):(.*)/, 2]
```

Comment:
first matching in regexp is useless. So I suggest to use `value[/table:.*:(.*)/, 1]`